### ISSUE TITLE
provide docker container and provide documentation

### ISSUE THREAD

**Issue by pahmann** [elisa-tech/meta-elisa#24]
Due to different build environments there was an alignment to have a docker image with the important tools to prepare the build available. This should be placed in a proper repository. 

Tasks are:
- Locate the old docker image
- Reproduce the build process using docker
- Document the steps so that other can follow
- Refer to the updated documentation in meta-elisa

**Comment by sudipm-mukherjee** [comment]
@pahmann  https://lists.elisa.tech/g/automotive/message/346

**Comment by pahmann** [comment]
@sudipm-mukherjee awesome. This gives a good start having documentation and dockerfile on the mailing list. Thank you.

**Comment by sudipm-mukherjee** [comment]
A documentation directory sounds good and then  you can place all the different ways to build in an organized way. Or else just a link to the mailing list link will also work. I dont know much about proper way of documenting, maybe @reiterative  might have an idea about how to properly store it.

**Comment by pahmann** [comment]
I sligthly modified the readme:
I added a section to place the docker file in your desired location where you want to start of with the docker image creation. I could not find this in the readme.
I had to run the commands for docker build and docker run with sudo rights. This may also require an update.
Also, I simply installed docker and docker.io from repo without adding the latest version as per docker documentation which is referred to in the readme. Maybe we could say something like use these packages and either follow the docker instructions, if this does not work for your distro or just give it a try with the few packages.

Next step can be to put the documentation in the meta-elisa/Documentation directory and refer to it within the readme.md as a good way to start the build. Especially, the script takes away some work for you.
